content/news/2021-XX-XX-fsf-statement.md

```diff
@@ -0,0 +1,25 @@
+title: "A Statement on Richard Stallman and the Free Software Foundation"
+authors: Owen Williams
+tags: community, diversity, politics
+date: 2021-03-26 00:00:00
+
+The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.
+His departure in 2019 was a consequence of the criticism he faced regarding his inappropriate behavior towards women, his hurtful comments about people with Down's Syndrome and his remarks relating to the Epstein scandal.
+His resignation came as a relief for many who have suffered his bad behavior.
+Richard Stallman's actions disqualify him from representing the FSF and the free software community as a whole.
+
+While Mixxx is not officially associated with the Free Software Foundation, there's an inevitable connection in the public's mind between all FOSS projects and the FSF.
+Furthermore, Mixxx is and always has been licensed under the terms of the GNU General Public License, version 2 or *later*, and only the Free Software Foundation can publish new versions of the GNU GPL which could be retroactively applied to the Mixxx code.
+Given this relationship, the Mixxx project has an interest in ensuring that the Free Software Foundation has inclusive, principled, and wise leadership which can effectively lead the broad free software community to meet present and future challenges.
+
+Therefore, we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation of Richard Stallman.
+The FSF board of directors, by allowing him to return, is giving the appearance of endorsing his conduct and can't remain effective without significant personnel changes and bylaw reforms.
+By sanctioning his return, the board of directors has lost all credibility on the issues of diversity, equity, and inclusion.
+We also call upon the Free Software Foundation to revise its bylaws to institute democratic processes for choosing leadership and increase the transparency of its operations.
```

Comment:
```suggestion
We acknowledge the Free Software Foundation's efforts to revise its bylaws to establish democratic processes for electing leadership and to increase the transparency of its activities.
```

Reply:
This reflects the current state better. We can't call something in a news addressing the Mixxx user.